CONTRIBUTING.md

```diff
@@ -102,15 +102,31 @@ Detailed explanation:
 
 8. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
     with a clear title and description.
+    
+    If page is not about standard Unix/Linux tool, please include link to the tool home page.
+    
+    If you are changing something non trivial, not just add page for new tool, please describe why are you doing this.
 
 9. Use Git's
    [interactive rebase](https://help.github.com/articles/interactive-rebase)
    feature to tidy up your commits before making them public.
+   
+   If you are not familiar with Git rebase, it might be helpful to check out these video tutorials:
+   - [Git Rebase: squash last commits](https://www.youtube.com/watch?v=qh9KtjfjzCU)
+   - [Learning Git Tutorial: Interactive Rebasing](https://www.youtube.com/watch?v=NW46XmvJh5Q)
+   
    In most cases it is better to squash commits before submitting a pull request.
 
-10. If you are asked to amend your changes before they can be merged in, please
-   use `git commit --amend` and force push to your remote feature branch.
-   You may also be asked to squash commits.
+10. If you do not want to do a rebasing, you can always overwrite your last commit in pull request. You can achieve this with `git commit --amend` command.
```

Comment:
Squashing can be done by rebase. It is the terms used by all tutorials and git help. That's what I ask them when I see 2+ commits in PR.

When there is one commit, it can be done without rebasing with `git commit --amend`